cancer_ai/validator/model_manager_test.py

```diff
@@ -44,20 +40,6 @@ def test_sync_hotkeys(model_manager: ModelManager):
     assert hotkey not in model_manager.get_state()
 
 
-def test_load_save_model_state(model_manager: ModelManager) -> None:
```

Comment:
do we have other function for this functionality?

Reply:
no, we don't serialize those objects, so not needed